backend/src/data_entry/validation.rs

```diff
@@ -436,6 +459,62 @@ impl Validate for PollingStationResults {
     }
 }
 
+impl Validate for ExtraInvestigation {
+    fn validate(
+        &self,
+        _election: &ElectionWithPoliticalGroups,
+        _polling_station: &PollingStation,
+        validation_results: &mut ValidationResults,
+        path: &FieldPath,
+    ) -> Result<(), DataError> {
+        if self.extra_investigation_other_reason.is_answered()
+            != self.ballots_recounted_extra_investigation.is_answered()
+        {
+            validation_results.errors.push(ValidationResult {
+                fields: vec![path.to_string()],
+                code: ValidationResultCode::F101,
+            });
+        }
+        if self.extra_investigation_other_reason.is_invalid()
+            || self.ballots_recounted_extra_investigation.is_invalid()
+        {
+            validation_results.errors.push(ValidationResult {
+                fields: vec![path.to_string()],
+                code: ValidationResultCode::F102,
+            });
+        }
```

Comment:
Does it make sense to show both F.101 and F.102, or should this be if-else if?

Reply:
For typist, the message is identical (except for the number of the error), so it does not add any value to show both of them. For coordinators, the message is different. 

I think this will occur more frequently, since we have updated the copy of the messages for typist to be a bit more generic (in most cases it is: "check if your answers align with the paper version"). We could implement some logic to only show multiple messages if the content actually differs, but I think that's a 'should have' and not a 'must have' for now